docs/application/tizen-studio/release-notes/3-7-release-notes.md

```diff
@@ -0,0 +1,55 @@
+# Tizen Studio 3.7 Release Notes
+
+- Release Date: Mar 25, 2020
```

Comment:
Please write extact release notes.
The releate date is different from that date of the toc.